core/main/src/Options/Interfaces/Particles/IParticles.ts

```diff
@@ -38,6 +39,7 @@ export interface IParticles {
     move: IMove;
     number: IParticlesNumber;
     opacity: IOpacity;
+    orbit: IOrbit;
```

Comment:
I see it under opacity? strange? whats missing here... @matteobruni

Reply:
Ok now it's fine